tool-timber/src/main/kotlin/com/infinum/sentinel/SentinelFileTree.kt

```diff
@@ -21,6 +22,8 @@ internal class SentinelFileTree(
     private val logFileResolver = LogFileResolver(context)
 
     override fun log(priority: Int, tag: String?, message: String, t: Throwable?) {
+        val dateTimeFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss. SSSZ", Locale.getDefault())
```

Comment:
If I'm interpreting the screenshot correctly, this space seems unnecessary.

```suggestion
        val dateTimeFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ", Locale.getDefault())
```